app/views/admin/partners/edit.html.erb

```diff
@@ -2,33 +2,34 @@
 <h1 class="page-header">Edit Partner: <em><%= @partner.name %></em></h1>
 
 <% if current_user.root? -%>
```

Comment:
this section of info is still hidden to everyone except root users - think we should probably just show all of this info to anyone who can view a partner in the admin!

Reply:
but it's not part of the issue A/C. perhaps i could extend/alter the issue to include all this stuff?

do you also want the ID readout?

Reply:
I think you wrote the AC! I'll add it in.